demo/algolia.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from algoliasearch.search_client import SearchClient
+
+class AlgoliaAPI(object):
+    """Class to connect to the Algolia API"""
+
+    def __init__(self, app_id, search_key, index):
+        self.client = SearchClient.create(app_id, search_key)
+        self.index = self.client.init_index(index)
+
+
+    def get_algolia_link(self, hits, index: int):
+        doc_link = f"- [{hits[index]['hierarchy']['lvl0']}"
+        if hits[index]['hierarchy']['lvl1']:
+            doc_link += f"/{hits[index]['hierarchy']['lvl1'].strip()}"
+            if hits[index]['hierarchy']['lvl2']:
+                doc_link += f"/{hits[index]['hierarchy']['lvl2'].strip()}"
+        doc_link += f"]({hits[index]['url']})"
```

Comment:
looks quite complicated! good work on this

Reply:
might be a little easier to understand if you store the different parts of the link in variables and then concatonate it at the end